parts/kuberneteswindowssetup.ps1

```diff
@@ -37,7 +37,7 @@ $global:AgentCertificate = "{{{clientCertificate}}}"
 $global:DockerServiceName = "Docker"
 $global:RRASServiceName = "RemoteAccess"
 $global:KubeDir = "c:\k"
-$global:KubeBinariesSASURL = "https://acsengine.blob.core.windows.net/v1-5-1/k.zip"
+$global:KubeBinariesSASURL = "https://acsengine.blob.core.windows.net/v1.5.3/k.zip"

```

Comment:
should be https://acsengine.blob.core.windows.net/v1-5-3/k.zip (azure storage does not allow '." in container names)